main.go

```diff
@@ -125,7 +140,7 @@ func init() {
 	viper.AutomaticEnv() // read in environment variables that match
 
 	viper.SetEnvPrefix("fn")
-	viper.SetDefault("api_url", "http://localhost:8080")
+	viper.SetDefault(envFnAPIURL, "http://localhost:8080")
 
 	// create aliases after api_url set
 	aliases = map[string]cli.Command{
```

Comment:
These can go back to where aliases is defined now that the building of the commands doesn't create the http client.